crypto/finite_field.ts

```diff
@@ -123,6 +123,9 @@ function createField(p: bigint, t: bigint, twoadicRoot: bigint) {
     t,
     twoadicRoot,
 
+    xor(x: bigint, y: bigint) {
+      return y ^ x;
+    },
```

Comment:
what I don't love about this function is that the output is not necessarily another field element (i.e. smaller than the modulus)

it seems a bit out of place in a finite field library. maybe we need a bigint library as well. maybe for another PR.

Reply:
sorry for being annoying but I don't like that either 😅 that behavior is unexpected. it's fine to leave the function here for now but better remove the `mod()`

Reply:
That's fine, I don't mind at all! I agree its an odd place for it to live in :D